CHANGELOG.md

```diff
@@ -0,0 +1,475 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
I don't think this is true

Reply:
@saschaisele-zf  if you are able to remove the semver statement i will merge it thanks :)

Reply:
Statement is gone